docs/docs/en/guide/task/switch.md

```diff
@@ -1,6 +1,7 @@
 # Switch
 
 The switch is a conditional judgment node, decide the branch executes according to the value of [global variable](../parameter/global.md) and the expression result written by the user.
+**Note** Execute expressions using javax.script.ScriptEngine.eval.
 
 ## Create
```

Comment:
```suggestion
## Create Task
```

Reply:
- Click `Project -> Management-Project -> Name-Workflow Definition`, and click the Create Workflow button to enter the DAG editing page.
- Drag from the toolbar <img src="/img/switch.png" width="20"/> to the canvas.